crates/bitwarden/src/sdk/request/fingerprint_request.rs

```diff
@@ -0,0 +1,11 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+pub struct FingerprintRequest {
+    /// The user's user id, used in the fingerprint generation process.
+    pub user_id: String,
```

Comment:
Might want to make this more generic `fingerprint_material` or something. This is used to generate fingerprint from a users email, instead of ID when authenticating with another device.